### DISCUSSION TITLE
Copilot - certificate is not yet valid

### DISCUSSION THREAD

**Discussion by salvatorpes** [community/community#61324] [Copilot Conversations] [answered]
### Select Topic Area

Bug

### Body

GitHub Copilot just stopped working yesterday. I've reinstalled it and also vscode. I signed out and back in on my github account and nothing happened. I've also confirmed that my subscription is still valid, which it still is. The error output is the following:

```
[INFO] [ghostText] [2023-07-16T21:19:27.317Z] Cancelled during debounce
[INFO] [default] [2023-07-16T21:19:27.436Z] [fetchCompletions] engine https://copilot-proxy.githubusercontent.com/v1/engines/copilot-codex
[ERROR] [default] [2023-07-16T21:19:27.566Z] Request Error: (FetchError) certificate is not yet valid
[ERROR] [ghostText] [2023-07-16T21:19:27.568Z] Error on ghost text request: (FetchError) certificate is not yet valid
```

I wish someone could help me with this,
Thank you

**Comment by Akash1134** [comment]
Hi @salvatorpes ,

Could you please install the [new IntelliJ release](https://plugins.jetbrains.com/plugin/17718-github-copilot/versions/stable/364783) and let us know if you continue to face this issue?

Thanks!

**Reply by salvatorpes** [reply]
Hi,

I use VScode and I don't think I'm able to install it.
How can I do that?

Thank you!!

**Comment by davecheney** [comment, marked as answer]
@salvatorpes we rotated our SSL certificate yesterday. The error you are seeing could indicate that the clock on your computer is not correct -- possibly you have your clock set to GMT, not your local time.

**Reply by salvatorpes** [reply]
My computer is set to UTC+0 which is equivalent to GMT. I'm currently in Lisbon so GMT is my local time. Should I still change the clock settings?

Thank you

**Reply by davecheney** [reply]
I'm sorry but I am not able to provide support for your laptop. The certificate is valid, if you are receiving a warning related to the validity date of the certificate it is possible this could be due to a local configuration on your machine.
```
* Server certificate:
*  subject: C=US; ST=California; L=San Francisco; O=GitHub, Inc.; CN=copilot-proxy.githubusercontent.com
*  start date: Jul 19 00:00:00 2023 GMT
*  expire date: Jul 19 23:59:59 2024 GMT
*  subjectAltName: host "copilot-proxy.githubusercontent.com" matched cert's "copilot-proxy.githubusercontent.com"
*  issuer: C=US; O=DigiCert Inc; CN=DigiCert Global G2 TLS RSA SHA256 2020 CA1
*  SSL certificate verify ok.
```

**Reply by davecheney** [reply]
@salvatorpes please double check your clock; your computer thinks it's the 16th.

**Reply by salvatorpes** [reply]
I'm so sorry. I was trying to change the date and time at the windows settings. 
My git account is logged in in ubuntu linux therefore I had to correct the date in the linux system not the windows one.
It was indeed wrong and I corrected it.
Thank you!!
